src/main.ts

```diff
@@ -107,11 +110,14 @@ export default class JumpToLink extends Plugin {
     getMode(currentView: View): VIEW_MODE {
         // @ts-ignore
         const isLegacy = this.app.vault.getConfig("legacyEditor")
+        const isLivePreview = !!currentView.containerEl.querySelector('.is-live-preview')
```

Comment:
使用 `cm6Editor.state.field(editorLivePreviewField)` 判断更合适。

Reply:
Thanks for the hint. I saw the `editorLivePreviewField` in the obsidian docs, but couldn't figure out how to use it.
However I have to look into it a bit, the `cmEditor` will be selected based on the `VIEW_MODE`, but this change would lead to `VIEW_MODE` depending on `cmEditor`.

Reply:
I changed it to use editorLivePreviewField